### PR TITLE
Docs overhaul

### PR DESCRIPTION
**Pull request by HowieG**
Resolves ENG-411

### CODE REVIEW

File: docs/v1/concepts/host-env.mdx [hunk 1]
```diff
@@ -33,6 +33,10 @@ Using an environment variable allows you to disable tracking of this data while
 can enable it again when deploying in a production environment.
 
 Opting out is not recommended by AgentOps, as it reduces information used to debug in the agent development process,
-[but AgentOps respects our privacy](mailto:alex@agentops.ai) conscious users and has made opting out available.
+but AgentOps respects our privacy conscious users and has made opting out available.
+
+<Warning>
+[Send us an email](mailto:alex@agentops.ai) with Subject: `Env Data Opt-Out`
```
Comment (siyangqiu) [inline docs/v1/concepts/host-env.mdx]: We don't have a way to opt users out right now, except via the SDK setting

**Review by siyangqiu — APPROVED** [review]

File: docs/v1/concepts/host-env.mdx (hunk 1 above)
Comment (HowieG) [reply docs/v1/concepts/host-env.mdx]: There was already an implied Call to Action to email Alex before this change lol

Comment (HowieG) [reply docs/v1/concepts/host-env.mdx]: I'll change it to just focus on the flag

Comment (HowieG) [reply docs/v1/concepts/host-env.mdx]: Done + broken links fixed

File: docs/v1/concepts/host-env.mdx [hunk 2]
```diff
@@ -23,16 +23,10 @@ and is never sold or shared.
...
 ## Opting-Out
 
-AgentOps users can opt out of certain environment data collection by setting an environment variable.
+AgentOps users can opt out of certain environment data collection by setting the following environment variable.
 
 ```
 AGENTOPS_ENV_DATA_OPT_OUT=True
 ```
 
-Using an environment variable allows you to disable tracking of this data while developing on a local machine. You
```
Comment (braelyn-ai) [inline docs/v1/concepts/host-env.mdx:32]: why remove this? it's a helpful comment to encourage a use case that we want

File: docs/v1/concepts/host-env.mdx [hunk 3]
```diff
@@ -23,16 +23,10 @@ and is never sold or shared.
...
-can enable it again when deploying in a production environment.
-
-Opting out is not recommended by AgentOps, as it reduces information used to debug in the agent development process,
-[but AgentOps respects our privacy](mailto:alex@agentops.ai) conscious users and has made opting out available.
-
-For more questions on privacy and data usage, please contact us on our official [Discord](https://discord.gg/UgJyyxx7uc).
```
Comment (braelyn-ai) [inline docs/v1/concepts/host-env.mdx:38]: i think we should leave our discord plug in

**Review by braelyn-ai — APPROVED** [review]
In the future, can we please keep PRs much much smaller. Even though these changes are all in the same vein, the larger the PR, the more likely stuff gets messed up, confused or lost

**Comment by HowieG** [comment]
@bboynton97 I needed to xlr8

**Comment by HowieG** [comment]
@bboynton97 but forreal I did one big PR bc I think it would've taken 3x as long to make branches for each and not have them conflict etc and the corresponding dip in motivation. Felt lower risk than code changes

**Comment by HowieG** [comment]
@bboynton97 otherwise yes I always try to keep my PRs modular

**Pull request merged**